spec/lib/workflow/stores/mongo-spec.js

```diff
@@ -323,33 +329,6 @@ describe('Task Graph mongo store interface', function () {
         });
     });
 
-    it('getActiveGraphById', function() {
```

Comment:
Why delete this test case? I didn't see you remove this function, so this unit-test need be kept.